python/setup.py

```diff
@@ -160,8 +163,12 @@ def write_version_py(filename=VERSION_PY_PATH):
     with open(os.path.join(filedir, 'README.rst')) as f:
         readme = f.read()
 
+    INSTALL_REQUIRES = []
     with open(os.path.join(filedir, 'requirements.txt')) as f:
-        INSTALL_REQUIRES = [i.strip() for i in f.readlines()]
+        INSTALL_REQUIRES += [i.strip() for i in f.readlines()]
+
+    with open(os.path.join(filedir, 'setup_requirements.txt')) as f:
+        INSTALL_REQUIRES += [i.strip() for i in f.readlines()]
```

Comment:
Do we want `setup_requirements.txt` in INSTALL_REQUIRES?  Does this make numba and friends a library dependency?  Is there an equivalent "build requirements"?

Reply:
I need to revisit this. I wasn't feeling I was quite done with the thought process with this. It's complicated to make a clean solution that caters for all the use cases.

Reply:
I think the main points are close to:
- with `sdist` package these requirements are needed as you are building `sbp` and the `parse_jit` module from the sources
- with `bdist_wheel` package these requirements are *not* needed but the wheel needs to be built for each Python version separately

Reply:
`sdist` is not uploaded to PyPi so I'll just add the version specific wheels and remove the `setup_requirements` from the package.